src/app/shared/markdown-wrapper/markdown-wrapper.component.ts

```diff
@@ -0,0 +1,36 @@
+import { AfterViewInit, Component, Input, OnInit, ViewChild } from '@angular/core';
+import DOMPurify from 'dompurify';
+import { MarkdownComponent } from 'ngx-markdown';
+
+@Component({
+  selector: 'markdown-wrapper',
+  templateUrl: './markdown-wrapper.component.html',
+  styleUrls: ['./markdown-wrapper.component.scss'],
+})
+
+// Wrapper for user-input markdown to provide custom sanitization not accessible through the built-in sanitizer, DOMSanitize.
```

Comment:
In so far as we do it, I think we tend to use TypeDoc style comments for class-level comments, e.g.,

```
/**
 Wrapper for...
 */
```

Reply:
Fixed.